src/doc/version-switcher.js

```diff
@@ -0,0 +1,210 @@
+const CURRENT_VERSION = "/* VERSION TO BE REPLACED */";
+
+function checkIfIsOldVersion() {
+    if (["http:", "https:"].indexOf(window.location.protocol) === -1) {
+        return;
```

Comment:
I am not a fan of JS returning undefined in some cases and a boolean value in others. Let's return false here.

Reply:
You're absolutely right. Changing it.